extra_script.py

```diff
@@ -83,36 +83,39 @@
     #   This solves a problem with duplicated symbols in Galactic
     global_env["_LIBFLAGS"] = "-Wl,--start-group " + global_env["_LIBFLAGS"] + " -l{} -Wl,--end-group".format(builder.library_name)
 else:
-    global_env['LIBS'].append(builder.library_name)
+    global_env.Append(LIBS=[builder.library_name])
 
 # Add library path
-global_env['LIBPATH'].append(builder.library_path)
+global_env.Append(LIBPATH=[builder.library_path])
 
 # Add required defines
-global_env['_CPPDEFFLAGS'] += ' -DCLOCK_MONOTONIC=0 '
+global_env.Append(CCFLAGS=['-DCLOCK_MONOTONIC=0'])
```

Comment:
Please use `global_env.Append(CPPDEFINES=[("CLOCK_MONOTONIC", 1)])`. 

A list of `(macro, value)` tuples.

Reply:
Addressed here: 8c2818bf6bce86a4b57b38b01db72642ec57cbd0

Reply:
Addressed here: 4b894e194f1a8e29379928f0f90c2f652e50a17a